### PR TITLE
Fix issues on volume snapshots page

### PR DESCRIPTION
**Pull request by ameade**
The annotations on a VolumeSnapshotContent is an object map, not a list.

### CODE REVIEW

**Comment by Ritikaa96** [comment]
Hi @ameade thanks for pointing out. 
Since we try to follow the [Trivial Edits](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits) guidelines as much as we can, I kindly ask you also look for further improvement on the same page so that one PR can cover more modifications. For eg. fixing other Grammatical errors or correcting broken links or more reference links etc.

File: content/en/docs/concepts/storage/volume-snapshots.md [hunk 1]
```diff
@@ -150,6 +155,11 @@ metadata:
 spec:
   source:
     volumeSnapshotContentName: test-content
+status:
+  boundVolumeSnapshotContentName: test-content
+  creationTime: "2023-02-20T02:41:12Z"
+  readyToUse: true
+  restoreSize: 1Gi
```
Comment (sftim) [inline content/en/docs/concepts/storage/volume-snapshots.md:162]: I wouldn't include a `status` here; it feels misleading to have it.

**Comment by sftim** [comment]
/retitle Fix issues on volume snapshots page

File: content/en/docs/concepts/storage/volume-snapshots.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/concepts/storage/volume-snapshots.md:162]: @ameade Sorry for the long delay, but would you mind removing `status` from this manifest? `status` is not part of the desired state of the VolumeSnapshot.

**Comment by kbhawkey** [comment]
👋 @ameade .
Are you actively working on these changes or can you close the pull request?
Thanks

File: content/en/docs/concepts/storage/volume-snapshots.md [hunk 2]
```diff
@@ -128,6 +128,11 @@ spec:
   volumeSnapshotClassName: csi-hostpath-snapclass
   source:
     persistentVolumeClaimName: pvc-test
+status:
+  boundVolumeSnapshotContentName: snapcontent-9ef06180-0665-4b1d-ae02-a1d35f74b262
+  creationTime: "2023-02-16T23:03:18Z"
+  readyToUse: true
+  restoreSize: 1Gi
```
Comment (xing-yang) [inline content/en/docs/concepts/storage/volume-snapshots.md:135]: I think "status" should not be included in a yaml file.  If you are showing an example of an already created VolumeSnapshot, then it is appropriate to show the status as well.  This is an example yaml file.

File: content/en/docs/concepts/storage/volume-snapshots.md (hunk 1 above)
Comment (xing-yang) [reply content/en/docs/concepts/storage/volume-snapshots.md:162]: Agreed that we should not include "status" in a yaml file example.

**Comment by Ritikaa96** [comment]
Hi @ameade please address the reviews suggested. Are you actively working this PR?

**Comment by natalisucks** [comment]
@ameade Hi there – due to the age of this PR and the lack of follow-up on reviews, I'll be closing this PR. Once you have the bandwidth, you are more than welcome to reopen this PR and continue the work, which will include checking out the feedback from our reviewers. Thanks!
/close

**Pull request closed without merging**